README.md

```diff
@@ -15,54 +15,56 @@ $ ansible-galaxy install -r requirements.yml
 
 ### Catalog
-* main.yml: Main playbook for Guest OS validation test
+* main.yml: Main playbook for guest operating system validation test
 * ansible.cfg: User customized Ansible configuration file
-* autoinstall: Folder for guest OS unattend install configuration files
+* autoinstall: Folder for guest operating system unattend install configuration files
 * common: Folder for common tasks called in test cases
 * docs: Folder for guide file and known issues
 * env_setup: Folder for playbooks or tasks which to prepare or cleanup testing environment
-* linux: Folder for playbooks to test Linux guest OS
-* windows: Folder for playbooks to test Windows guest OS
+* linux: Folder for playbooks to test Linux guest operating system
+* windows: Folder for playbooks to test Windows guest operating system
 * plugin: Folder for plugin scripts
 * tools: Folder for 3rd-party tools used in test cases
 * vars: Folder for variable files used in testing
 * changelogs: Folder for changelog of each release 
 
 ### Supported Testing Scenarios
-This project supports below scenarios for end-to-end guest OS validation testing 
-* Deploy VM and install guest OS from ISO image
+This project supports below scenarios for end-to-end guest operating system validation testing 
+* Deploy VM and install guest operating system from ISO image
 * Deploy VM from an OVA template
-* Existing VM with installed guest OS, which should satisfy below requirments.
-  * SSH and Python are installed and enabled
-  * The vm_python variable in vars/test.yml must be set with correct python path. Or user can set PATH in /etc/environment in guest OS to include the binary directory path to python.
-  * The root user should be enabled and permitted to log in through SSH in Linux guest OS
-  * Execute [ConfigureRemotingForAnsible.ps1](https://github.com/ansible/ansible/blob/devel/examples/scripts/ConfigureRemotingForAnsible.ps1) script in Windows guest OS in advance
+* Existing VM with installed guest operating system, which should satisfy below requirments.
+  * SSH and Python are installed and enabled.
+  * The vm_python variable in vars/test.yml must be set with correct python path. Or user can set PATH in /etc/environment in guest operating system to include the binary directory path to python.
+  * The root user should be enabled and permitted to log in through SSH in Linux guest operating system.
+  * Execute [ConfigureRemotingForAnsible.ps1](https://github.com/ansible/ansible/blob/devel/examples/scripts/ConfigureRemotingForAnsible.ps1) script in Windows guest operating system in advance.
 
-### Supported Guest OS
+### Compatible Guest Operating Systems
 
-| Guest OS types/versions                         | Automatic install from ISO image | Deploy from ova template | Existing VM and installed guest OS |
+| Guest Operating System Types/Versions                         | Automatic Install from ISO Image | Deploy from OVA Template | Existing VM And Installed Guest Operating System |
```

Comment:
Or we can just use "Guest Operating System Versions". Does "And" need to be captitalized?

Reply:
Updated `Guest Operating System Types/Versions` to `Guest Operating Systems` and updated `Existing VM And Installed Guest Operating System` to `Existing VM with Guest Operating System Installed`.